### DISCUSSION TITLE
Prisma Multiple Database

### DISCUSSION THREAD

**Discussion by k2nneth** [prisma/prisma#12638] [Q&A]
Any guidelines to connect multiple databases for NestJS + Prisma?

**Comment by austincrim** [comment]
Hey @k2nneth 👋 ,

Multiple databases are not yet tightly integrated into Prisma, however you can take a look at [this comment](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) for a couple usable workarounds. Feel free to +1 that issue to help us prioritize.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
